frontend/src/components/LegalNotice/LegalNotice.jsx

```diff
@@ -33,57 +35,45 @@ const LegalNotice = () => {
             <br />
             SIREN : 120 001 011
           </p>
-        </div>
-        <div>
-          <h3>Directeur de la publication</h3>
-          <p className="fr-mb-2w">
+        </Section>
+        <Section title="Directeur de la publication">
+          <span>
             MINISTÈRE DE L&lsquo;ENSEIGNEMENT SUPÉRIEUR,
             DE LA RECHERCHE ET DE L&lsquo;INNOVATION
-          </p>
-          <p className="fr-mb-2w">1 rue Descartes - 75231 Paris cedex 05</p>
-        </div>
-        <div>
-          <h3>Hébergement de la Plateforme</h3>
+          </span>
+          <p>1 rue Descartes - 75231 Paris cedex 05</p>
+        </Section>
+        <Section title="Hébergement de la Plateforme">
           Ce site est hébergé en propre par Scalingo SAS, 15 avenue du Rhin, 67100 Strasbourg, France.
-        </div>
-        <div id="accessibilite">
-          <h3>Accessibilité</h3>
-          <p className="fr-mb-2w">
+        </Section>
+        <Section title="Accessibilité">
```

Comment:
Add `id` prop to `Section` to be able to add `id="accessibilite"` here (it's used by the footer "Accessibilité" link)